querybook/webapp/lib/python/usePython.ts

```diff
@@ -108,10 +144,17 @@ export default function usePython({
         [setStdout, setStdout, runPython, docId, stdoutCallback, stderrCallback]
     );
 
+    const cancelRunPython = useCallback(async (): Promise<void> => {
```

Comment:
what's the point of async?

Reply:
cancel will not actually happen immediately after setting the signal buffer. This is to simulate it to be a promise and resolve after the cancel event actually happens.

Reply:
there is no await here so I dont think the async does anything?

Reply:
oh yeah, correct. the "aync" is not needed.